### DISCUSSION TITLE
Trigger backports with PR description

### DISCUSSION THREAD

**Discussion by EduPonz** [Mergifyio/mergify#4206] [Q&A]
Hi everyone,

First of all, thanks for your great work! At eProsima/Fast-DDS, we use the Mergify app to trigger backports by writing a comment in the PR, which works great!

We have recently incorporated a more complete PR template and we have tried to trigger the backports from the PR description body itself, which seems not supported at the moment. Am I correct? This would be very useful for us since we already consider which PRs would need to be backported before issuing them, what are your thoughts?

Thanks!

**Comment by jd** [comment]
Hi @EduPonz 

You're right this is not supported. I'm not sure this is something we'd add short term as the PR body is most often used as, e.g., a commit template, rather than as a command center.

- You can however backport on multiple branches with a single comment like `@mergifyio backport foo bar`
- If the target branch for backports are not changing too often you could rules + labels to trigger the backport when creating the PR
